### DISCUSSION TITLE
No actions button in the web UI

### DISCUSSION THREAD

**Discussion by fraca7** [buildbot/buildbot#8010] [Q&A] [answered]
Hello. I'm in the process of "porting" an old 2.x configuration to 4.0.2. It's a simple setup: no auth (it's done upstream by nginx), single repository (mercurial, polled), inrepo branches. My current problem is that the "Actions" button that should appear in the workers list and on each worker page is just missing, so I can't graceful shutdown a worker.

Am I missing something here? I searched the manual and googled for hours to no avail.

**Comment by p12tic** [comment, marked as answer]
Please file a bug with a screenshot.

**Reply by fraca7** [reply]
Thanks, done: #8063
